demos/chips.html

```diff
@@ -38,65 +38,129 @@
     <main class="mdc-toolbar-fixed-adjust">
       <section class="hero">
         <div class="mdc-chip-set">
-          <div class="mdc-chip">
+          <div class="demo-chip mdc-chip" tabindex="0">
```

Comment:
We've given the trailing icons `role`, but what about chips themselves? (I forget if that's being handled in #2215)

Reply:
I think @bonniezhou decided to do most of the a11y stuff (the tricky parts, anyway) in a separate follow up PR, cuz shoehorning it into this one was getting kinda hairy.

Reply:
Yep, it's tracked in #2259. Also, design confirmed that we should add a `role` but looking into options other than `button` (maybe `menuitem` or something similar).